### PR TITLE
Update jdk10

### PR DESCRIPTION
**Pull request by jaromil**
clojure 10 and travis new targets for jdk10

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,9 +9,8 @@ before_script:
   - mongo --version
 script: lein midje
 jdk:
-  - openjdk7
-  - oraclejdk8
   - openjdk8
-  - oraclejdk9
```
Comment (aspasia) [inline .travis.yml]: why remove oraclejdk9????

File: .travis.yml [hunk 2]
```diff
@@ -9,9 +9,8 @@ before_script:
   - mongo --version
 script: lein midje
 jdk:
-  - openjdk7
-  - oraclejdk8
```
Comment (aspasia) [inline .travis.yml]: why remove oraclejdk8?

File: project.clj [hunk 3]
```diff
@@ -7,7 +7,11 @@
             :year 2017
             :key "gpl-3.0"}
 
-  :dependencies [[org.clojure/clojure "1.9.0"]
+  ;; fix for jdk 10
+  :managed-dependencies [[org.clojure/core.rrb-vector "0.0.14"]
```
Comment (aspasia) [inline project.clj]: what are those needed for?

Comment (aspasia) [reply project.clj]: Please share link with suggested fix. Was it failing without it?

Comment (jaromil) [reply project.clj]: was link you also mentioned best source for upgrade

**Comment by jaromil** [comment]
sry no time to follow up also closing

**Pull request closed without merging**